src/modules/admin/components/formRules/FormRule.tsx

```diff
@@ -51,69 +70,87 @@ const FormRule: React.FC<Props> = ({
     />
   );
 
-  const conditions = [];
-
-  if (projectType)
-    conditions.push(
-      <FormRuleChip
-        label={`Project Type is ${HmisEnums.ProjectType[projectType]}`}
-      />
-    );
-  if (project)
-    conditions.push(
-      <FormRuleChip label={`Project is ${project.projectName}`} />
-    );
-  if (organization)
-    conditions.push(
-      <FormRuleChip
-        label={`Organization is ${organization.organizationName}`}
-      />
-    );
-  if (otherFunder) {
-    conditions.push(
-      <FormRuleChip label={`Funding Source is ${otherFunder}`} />
-    );
-  } else if (funder) {
-    conditions.push(
-      <FormRuleChip
-        label={`Funding Source is ${HmisEnums.FundingSource[funder]}`}
-      />
-    );
+  const conditions: Record<string, ReactNode> = {};
```

Comment:
@gigxz I'd especially appreciate your review on the changes here. I've tried to add back functionality that I lost in implementing the new Form Rule table (in the [previous PR](https://github.com/greenriver/hmis-frontend/pull/826/files#diff-263242b358a931201dbc10b01b439b9a544c1ec5bbc1cd611d00dfce7a30534dL42-L97)).

One specific thing I'm worried about is ending up with a form rule that's displayed as a blank row in the table. I've tried to protect against that [here](https://github.com/greenriver/hmis-frontend/pull/829/files#diff-9ea65162911b6860febdc61e834e3f30615253867cc120610e61b9c20cad815eR126-R132).

Reply:
Thanks for the call-out. I played around with it a little and have a snippet suggestion below. It includes:

1) More logical display for service type field. The service fields indicate which service type(s) this form collects, so it doesn't really make sense to display it as a "condition". If you look at the default HUD service form you can see lots of examples (screencapped below).
<img width="1181" alt="Screenshot 2024-07-03 at 4 59 25 PM" src="https://github.com/greenriver/hmis-frontend/assets/5333982/8b9ae329-d188-4ef7-b0ff-6718cc2a6d87">

2) Show as "Applies to all clients in all projects" if there are no project conditions selected.:
<img width="411" alt="Screenshot 2024-07-03 at 5 07 47 PM" src="https://github.com/greenriver/hmis-frontend/assets/5333982/f5107a35-d77f-4c41-bce1-26acba737550">


```
  // remove service stuff from "conditions". 

  const serviceCollected =
    formRole === FormRole.Service && (serviceType || serviceCategory) ? (
      <>
        Collects{' '}
        <FormRuleChip
          label={serviceType?.name || serviceCategory?.name || 'Service'}
        />
      </>
    ) : null;


  return (
    <Stack direction='row' gap={2}>
      <Typography variant='body2' sx={{ flexGrow: 1 }}>
        {serviceCollected}
        {!nonClient && (
          <>
            {serviceCollected ? ' for' : 'Applies to'} {appliesTo}
          </>
        )}
        {conditionCount === 0 &&
          (nonClient ? (
            <>
              Applies to <FormRuleChip label='All Projects' />
            </>
          ) : (
            <>
              {' '}
              in <FormRuleChip label='All Projects' />
            </>
          ))}

        {conditionCount > 0 && (
          //... etc.
```

Reply:
Thanks Gig this is great. I refactored it a little for clarity/simplicity, but I think I've checked each case and kept the functionality from your snippet.